orbit/pkg/table/firmware_eficheck_integrity_check/firmware_eficheck_integrity_check_darwin.go

```diff
@@ -0,0 +1,77 @@
+//go:build darwin
+// +build darwin
+
+// Package firmware_integrity_check implements a table
+// to perform an integrity check for Legacy EFI.
+package firmware_eficheck_integity_check
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/osquery/osquery-go/plugin/table"
+	"github.com/rs/zerolog/log"
+	"golang.org/x/sys/unix"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.TextColumn("chip"),
+		table.TextColumn("output"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+//
```

Comment:
nit: no need for extra comment lines

Reply:
Not following, why not?

Reply:
I see we have two styles in our code. one with space lines and one without.
It's a nit pick issue. we can ignore it for now.